contracts/PaymentReceiver.sol

```diff
@@ -4,7 +4,7 @@ import "hardhat/console.sol";
 import "./IPaymentReceiver.sol";
 
 contract ClientDatabase {
-    uint private nonce;
+    uint private nextId;
```

Comment:
https://docs.openzeppelin.com/contracts/4.x/api/utils#Counters

You might want to consider using Counters as a way to restrict _id from being increased more than 1